frontend/src/styles/_theme.scss

```diff
@@ -81,3 +81,10 @@ body {
     color: $secondary-1;
   }
 }
+
+@mixin add-background-image {
```

Comment:
@farhaanbukhsh, nit: probably `add-header-background` (or `set-header-background`) is a better name?

Reply:
Done